src/status_im/ui2/screens/chat/home.cljs

```diff
@@ -0,0 +1,373 @@
+(ns status-im.ui2.screens.chat.home
```

Comment:
this file doesn't exist anymore , see https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/chat/home/view.cljs